lamby/lamby.py

```diff
@@ -44,11 +50,112 @@ def uninit():
 @click.argument('files', nargs=-1)
 @click.option('-m', '--message')
 def commit(files, message):
-    click.echo('Commiting following files:')
+    lamby_dir = './.lamby'
+    if not os.path.isdir(lamby_dir):
+        click.echo('Lamby project has not been initialized in ' + os.getcwd())
+        sys.exit(1)
+
+    if len(files) == 0:
+        files = search_file_type('.', 'onnx')
+
+    log = deserialize_log()
+
     for file in files:
-        click.echo('\t' + file)
-    if message is not None:
-        click.echo('Commit Message: ' + message)
```

Comment:
I liked these outputs but don't see them anywhere. Any plans to add them back in? Not necessarily here but I think this message was useful

Reply:
They were there as placeholder, but I agree, they would be a good addition.